app/AttributionGenerator.js

```diff
@@ -158,7 +158,9 @@ $.extend( AttributionGenerator.prototype, {
 		// Prefer attribution over author:
 		var $attribution = this._asset.getAttribution();
 		if( $attribution ) {
-			return format === 'html' ? $attribution : $attribution.text();
+			if( $attribution.length && $attribution.text().length ) {
```

Comment:
Ooops, I didn't think about it. But it does definitely make sense to be explicit there, of course.

I quickly looked at code in this repo and there are both approaches mixed. Then explicit checking the integer seems to be slightly more often used. I will make some tour through the code to make it more consistent. Thanks for pointing this out.

@Andrew-WMDE sorry for pointing you in the wrong direction!